### PR TITLE
feat(onpremise): Support custom CA roots in Docker image

### PR DESCRIPTION
**Pull request by Spice-King**
Trigger updating the system CA roots if there are local certificates,
and add env vars to override some bundled CA roots.

Result of poking and prodding from getsentry/sentry#26851
~~Required for getsentry/onpremise#1015~~

### CODE REVIEW

File: docker/docker-entrypoint.sh [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 set -e
 
+if [ "$(ls -A /usr/local/share/ca-certificates/)" ]; then
+  update-ca-certificates1
```
Comment (BYK) [inline docker/docker-entrypoint.sh:6]: Is this a typo or the command name really ends with a `1`?
```suggestion
  update-ca-certificates
```

**Review by BYK — CHANGES_REQUESTED** [review]
Looks good to me, just need to make sure the command is correct.

File: docker/docker-entrypoint.sh (hunk 1 above)
Comment (Spice-King) [reply docker/docker-entrypoint.sh:6]: Typo, though how the hell did that make it in there? I did test the changes before I made the commits.

**Review by BYK — APPROVED** [review]

**Comment by BYK** [comment]
We have decided to isolate this feature and logic over at the onpremise repo as the dynamic volume mount and init in entrypoint are not something everybody might choose (over building their own image for instance).

See getsentry/onpremise#1015 for more details.

**Pull request closed without merging**